tests/simulator_test.cpp

```diff
@@ -64,48 +66,57 @@ class EnergyCollectorTest : public ::testing::Test {
     return false;
   }
 
-  void
-  printCollectors(const std::vector<std::unique_ptr<objects::EnergyCollector>>
-                      &energyCollectors) const {
+  float getCollectorRadius(const Collectors &energyCollectors) const {
+    return energyCollectors[0]->getRadius();
+  }
+
+  // returns value of angle between two neighborhood collectors and
+  // Vec3(0, 0, 0)
+  float getCollectorAngle(const Collectors &energyCollectors) const {
```

Comment:
Why do you pass entire collector vector here?

Reply:
What? Why do you need to pass collectors at all?

Reply:
LOL, to less coffee I guess. fixed ;)